localstack/utils/aws/aws_stack.py

```diff
@@ -546,6 +554,12 @@ def send_event_to_target(arn, event):
         LOG.info('Unsupported Events rule target ARN "%s"' % arn)
 
 
+def get_events_target_attributes(target):
+    # added for sqs, if needed can be moved to an if else
+    # block for multiple targets
+    return target.get('SqsParameters')
```

Comment:
Which other parameter types could occur here? Wondering if we should add the remaining ones as part of this PR as well..

Reply:
KinesisParameters
RunCommandParameters
EcsParameters
BatchParameters
SqsParameters
HttpParameters
HeaderParameters
QueryStringParameters
RedshiftDataParameters